src/Revalidation.js

```diff
@@ -242,10 +229,12 @@ function revalidation(
       const valid = isValid(validate(rules, form)) &&
         isValid(errors) &&
         (validateOnChangeResult && isValid(asyncErrors))
+      const required = validate(rules, initializeValue('', form))
```

Comment:
@busypeoples this runs rules against an empty form. This may not work as desired for conditionally required fields since the condition could be the value of another field. I think that would required a "simulated value" feature in spected that could only change the current rule target's value to the simulated value (in our case empty string).

Reply:
Been thinking about the problem now. Yes, if we want to make sure that the calculated `required` is independent from the actual value, but dependent on other fields (in case of depending fields) then we have to solve this outside of `revalidation`. We would have to calculate the `required` on every form update, as the dependent fields might have changed.